dvc/command/plot.py

```diff
@@ -0,0 +1,228 @@
+import argparse
```

Comment:
@pared @jorgeorpinel 

please:

- [ ] review CLI options, help, etc
- [ ] adjust completion scripts?

Reply:
>  review CLI options, help, etc

Done in https://github.com/iterative/dvc/pull/3577#pullrequestreview-401499021